extensions/amazon-lambda-resteasy/deployment/src/main/java/io/quarkus/amazon/lambda/resteasy/deployment/AmazonLambdaResteasyProcessor.java

```diff
@@ -22,4 +41,61 @@ void setup(AmazonLambdaResteasyRecorder recorder, Optional<ResteasyServerConfigB
             recorder.initHandler(resteasyServerConfig.get().getInitParameters(), config);
         }
     }
+
+    @BuildStep
+    @Record(ExecutionTime.STATIC_INIT)
+    void sam() {
+        if (config.updateConfig) {
```

Comment:
I think this will need to be updated after https://github.com/quarkusio/quarkus/pull/3338 goes in? It seems like ideally we would be outputing a zip with the template and the app?

Reply:
in this case, the amazon SAM plugin already does its own packaging and just run `compile`/`package` and then bundles things up.  There's not *that* much here to do on the quarkus side other than our normal bits.